### ISSUE TITLE
Add distilbert and quantized pytorch models to the neuspell catalogue

### ISSUE THREAD

**Issue by murali1996** [neuspell/neuspell#25]
- Follow <https://pytorch.org/tutorials/intermediate/dynamic_quantization_bert_tutorial.html#apply-the-dynamic-quantization> for quantization code.
- train a distilbert (https://huggingface.co/distilbert-base-cased)
- Evaluate performance (time and accuracy) of the test sets (natural)
- Update README.md

**Comment by seraconlp** [comment]
Good idea!
